Module4_Labs/Lab2_Doubly_Linked_List/Cards/81.md

```diff
@@ -0,0 +1,9 @@
+Remember that Big O notation describes the behavior of functions as input grows, in this case as the number of elements in the doubly linked lists increases. 
+
+The time complexity O(n) describes the behavior of the access and search functions on a doubly linked list. Here, **n** refers to the size of the input. So the greater the input, in this case the number of elements in the list, the more time these operations will take.
```

Comment:
can you explain why? Like how the data structure (which is comprised of pointers) makes you do this?